libs/blocks/comparison-table/comparison-table.css

```diff
@@ -0,0 +1,490 @@
+  border-bottom: 1px solid var(--comparison-row-border-color);
+}
+
+.comparison-table .table-container .table-row-header {
+  text-align: start;
+  font-size: var(--type-body-xs-size);
+  font-style: normal;
+  font-weight: var(--type-detail-all-weight);
+  line-height: 130%;
+  padding: var(--spacing-s) var(--spacing-s) var(--spacing-s) 0;
+  display: flex;
+  align-items: center;
+  justify-content: flex-start;
+  margin-inline-end: 12px;
+}
+
+.comparison-table .table-container .table-row-header a {
+  display: contents;
+}
+
+.comparison-table span[data-tooltip] .icon-milo:hover {
+  cursor: pointer;
+}
+
+.comparison-table span[data-tooltip] .icon-milo:hover path {
+  color: var(--hover-border-color);
+}
+
+.comparison-table .table-container .primary-cell .icon-checkmark path {
+  color: var(--primary-cell-path-color);
+}
+
+.comparison-table .table-container .table-row .table-cell:nth-child(odd) div {
+  background-color: var(--comparison-column-background-color);
+}
+
+.comparison-table .table-container .table-cell {
+  padding: 36.5px 0;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.comparison-table .table-container .table-cell p {
+  text-align: center;
+  min-height: 18px;
+  width: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  word-break: break-word;
+  font-size: var(--type-body-xs-size);
+  line-height: var(--type-body-xs-lh);
+}
+
+.comparison-table .table-container .table-cell div {
+  border: 1px solid var(--comparison-border-color);
+  border-radius: 4px;
+  width: calc(100% - 26px);
+  padding: var(--spacing-xs) 12px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.comparison-table .table-container .table-cell >p:nth-child(2) {
+  margin-top: 4px;
+  font-size: var(--type-body-xxs-size);
+  font-weight: 400;
+  line-height: var(--type-body-xxs-lh);
+}
+
+.comparison-table *:is(p, h1, h2, h3, h4, h5, h6) {
+  margin: 0;
+  display: flex;
+  align-items: center;
+}
+
+.comparison-table .milo-tooltip svg {
+  height: 12px;
+  width: 12px;
+}
+
+.comparison-table .header-content {
+  padding-bottom: var(--spacing-s);
+}
+
+.comparison-table .header-content.sticky {
+  position: sticky;
+  top: 0;
+  z-index: 9;
+  background-color: var(--comparison-column-background-color);
+  width: 100%;
+  box-shadow: 0 1px 6px 0 rgba(0, 0, 0, 0.12);
+}
+
+.comparison-table .header-content.sticky .header-content-wrapper {
+  justify-content: space-between;
+}
+
+.comparison-table .header-content-wrapper {
+  max-width: calc(var(--comparison-desktop-max-width) - 2*var(--comparison-header-spacing) - 2px);
+  margin: 0 auto;
+  display: grid;
+  grid-template-columns: minmax(268px, 1.15fr) repeat(auto-fit, minmax(100px, 1fr));
+  gap: 12px;
+}
+
+.comparison-table[data-child-count="3"] .header-content-wrapper {
+  grid-template-columns: minmax(268px, 1fr) repeat(auto-fit, minmax(100px, 1fr));
+}
```

Comment:
Would it make sense to group this along with the one on L91, since the declaration is identical?